.github/workflows/build.yml

```diff
@@ -3,42 +3,23 @@ on:
   push:
     branches:
       - master
-permissions:
-  id-token: write
-  contents: write
+      - test-reusable-workflows
```

Comment:
Should this be left in here?

Reply:
Nope, thanks. that's just for testing, I've deleted it.